src/tests/JIT/Directed/Convert/out_of_range_fp_to_int_conversions.cpp

```diff
@@ -137,6 +137,7 @@ extern "C" DLLEXPORT  uint64_t ConvertDoubleToUInt64(double x, FPtoIntegerConver
         return ((x != x) || (x < INT64_MIN) || (x >= uint64_max_plus_1)) ? (uint64_t)INT64_MIN : (x < 0) ? (uint64_t)(int64_t)x : (uint64_t)x;
 
     case CONVERT_SENTINEL:
+    case CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64:
```

Comment:
This behavior under this case is not the backward compatible (ie .NET Framework) behavior. 

The `CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64` case should stay as is; or the enum `CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64` should be renamed to match its new meaning.

Reply:
I think it will be better to rename the enum since we are now changing it to match the corresponding AVX512 instruction vcvttsd2usi. I will be changing the enum name.